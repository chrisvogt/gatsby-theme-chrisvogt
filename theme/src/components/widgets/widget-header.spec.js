import React from 'react'
import renderer from 'react-test-renderer'
import WidgetHeader from './widget-header'

const aside = (
  <div className='sidebar-content'>
    Sidebar
  </div>
)

describe('WidgetHeader', () => {
  it('matches the snapshot', () => {
    const widgetTitle = 'Neat & Interesting Widget'
    const tree = renderer
      .create(<WidgetHeader aside={aside} platform='GitHub'>{widgetTitle}</WidgetHeader>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
