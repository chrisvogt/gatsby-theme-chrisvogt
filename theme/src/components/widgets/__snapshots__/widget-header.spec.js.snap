// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetHeader matches the snapshot 1`] = `
<header
  className="css-1gotcft-WidgetHeader"
>
  <h2
    className="css-1wf7gym"
  >
    Neat & Interesting Widget
  </h2>
  <div
    className="css-idc44k-WidgetHeader"
  >
    <div
      className="sidebar-content"
    >
      Sidebar
    </div>
  </div>
</header>
`;
